packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -2003,6 +2004,43 @@ describe('function', () => {
       });
     });
   });
+
+  describe('code signing config', () => {
+    test('default', () => {
+      const stack = new cdk.Stack();
+
+      const signingProfile = new signer.SigningProfile(stack, 'SigningProfile', {
+        platform: signer.Platform.AWS_LAMBDA_SHA384_ECDSA,
+      });
+
+      const codeSigningConfig = new lambda.CodeSigningConfig(stack, 'CodeSigningConfig', {
+        signingProfiles: [signingProfile],
+      });
+
+      new lambda.Function(stack, 'MyLambda', {
+        code: new lambda.InlineCode('foo'),
+        handler: 'index.handler',
+        runtime: lambda.Runtime.NODEJS_10_X,
+        codeSigningConfig,
+      });
+
+      expect(stack).toHaveResource('AWS::Lambda::Function', {
+        Properties: {
+          Code: { ZipFile: 'foo' },
+          Handler: 'index.handler',
+          Role: { 'Fn::GetAtt': ['MyLambdaServiceRole4539ECB6', 'Arn'] },
+          Runtime: 'nodejs10.x',
+          CodeSigningConfigArn: {
+            'Fn::GetAtt': [
+              'CodeSigningConfigD8D41C10',
+              'CodeSigningConfigArn',
+            ],
+          },
+        },
+        DependsOn: ['MyLambdaServiceRole4539ECB6'],
+      }, ResourcePart.CompleteDefinition);
```

Comment:
code simplification: 

```suggestion
      expect(stack).toHaveResource('AWS::Lambda::Function', {
        CodeSigningConfigArn: {
          'Fn::GetAtt': [
            'CodeSigningConfigD8D41C10',
            'CodeSigningConfigArn',
          ],
        },
      });
```